<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>BaseMenuDialogUi</class>
  <widget class="QDialog" name="BaseMenuDialogUi">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>360</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin"><number>10</number></property>
      <property name="topMargin"><number>0</number></property>
      <property name="rightMargin"><number>10</number></property>
      <property name="bottomMargin"><number>10</number></property>
      <item>
        <widget class="QFrame" name="customTitleBar">
          <property name="frameShape"><enum>QFrame::NoFrame</enum></property>
          <layout class="QHBoxLayout" name="customTitleLayout">
            <property name="leftMargin"><number>0</number></property>
            <property name="topMargin"><number>0</number></property>
            <property name="rightMargin"><number>0</number></property>
            <property name="bottomMargin"><number>10</number></property>
            <item>
              <widget class="QLabel" name="customTitle">
                <property name="text"><string>Dialog</string></property>
              </widget>
            </item>
            <item>
              <spacer name="titleSpacer">
                <property name="orientation"><enum>Qt::Horizontal</enum></property>
                <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
              </spacer>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignTop">
              <widget class="QPushButton" name="customCloseBtn">
                <property name="text"><string>Ã—</string></property>
                <property name="minimumSize"><size><width>28</width><height>28</height></size></property>
                <property name="maximumSize"><size><width>28</width><height>28</height></size></property>
                <property name="flat"><bool>true</bool></property>
                <property name="toolTip"><string>Close</string></property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QWidget" name="contentContainer">
          <layout class="QVBoxLayout" name="contentLayout">
            <property name="leftMargin"><number>0</number></property>
            <property name="topMargin"><number>0</number></property>
            <property name="rightMargin"><number>0</number></property>
            <property name="bottomMargin"><number>0</number></property>
            <item>
              <spacer name="contentSpacer">
                <property name="orientation"><enum>Qt::Vertical</enum></property>
                <property name="sizeHint" stdset="0"><size><width>20</width><height>20</height></size></property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
